x-pack/plugins/drilldowns/public/index.ts

```diff
@@ -7,12 +7,14 @@
 import { DrilldownsPlugin } from './plugin';
 
 export {
-  DrilldownsSetupContract,
-  DrilldownsSetupDependencies,
-  DrilldownsStartContract,
-  DrilldownsStartDependencies,
+  SetupContract as DrilldownsSetupContract,
```

Comment:
In the spirit of consistency, should just be `DrilldownsSetup` (and the others) see https://github.com/elastic/kibana/issues/60295

Reply:
Done.